src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/synthetic_image_generator.py

```diff
@@ -50,9 +51,13 @@ def create_synthetic_image(
         image_width_stddev: int,
         image_height_mean: int,
         image_height_stddev: int,
-        image_format: ImageFormat,
+        image_format: Optional[ImageFormat] = None,
     ) -> str:
         """Generate base64 encoded synthetic image using the source images."""
+        if image_format is None:
+            image_formats = list(ImageFormat)
+        else:
+            image_formats = [cast(ImageFormat, image_format)]
```

Comment:
Can we have a utility function like below for getting enum members under `genai_perf/utils.py`:
```python
def get_enum_members(enum: Enum) -> List[Any]:
    return list(enum.__members__.values())
```

and use that to randomly select image format?
```suggestion
        if image_format is None:
            image_format = random.choice(utils.get_enum_members(ImageFormat))
```

Reply:
Makes more sense. Thanks, I will update shortly.

Reply:
Actually we need the name of enum members, not the value. So I think keeping the variable in ImageFormat type is cleaner. Please check if you are fine with the 49b818a3be92b6b071a07efca219f0b2ce0c708d, or you prefer creating dedicated function in util module for that.

Reply:
Oh I think the commit looks good 👍